src/components/SecondaryPanes/Expressions.js

```diff
@@ -40,13 +41,15 @@ function getValue(expression) {
 }
 
 const Expressions = React.createClass({
+  _input: (null: any),
```

Comment:
I'm confused here. It looks like you are adding `_input` to the prototype because you're setting it to null. You should be able to only do the type definition here, without actually setting a value and the error should go away. Flow treats class definitions as invariants.

```
const Expression = React.createClass({
  _input: ?HTMLInputElement,
```